structure/0018-technical-teams.md

```diff
@@ -1,3 +1,16 @@
+# This Document is Deprecated
+
+[Technical Leads](#technical-leads) have been removed in favor of [Team Leads](0006-governance.md#team-leads).
+
+[Team Responsibilites](#team-responsibilities) have been generalized for all teams in [Team Responsibilities](0006-governance.md#team-responsibilities).
+
+[Technical Teams](#technical-teams) is outdated, but *all* teams can be found at https://github.com/orgs/QuiltMC/teams.
```

Comment:
oh, right. uh.....
I assume there's no way to fix this from github?

Reply:
Sadly not, people can choose to not be public members of the org.

Reply:
i don't really care about seeing the members, but seeing the teams would be good.

Reply:
I'm a member of the org and I can't even see it

Reply:
Scratch that, there's a . at the end lmao

Reply:
To see teams publicly, you must set this option for every team:

https:\/\/github.com/orgs/QuiltMC/teams/\<TEAM\>/edit
![image](https://user-images.githubusercontent.com/30361266/155811387-4ebd89ca-80a2-4553-9d76-a52d36ea9e9d.png)

Reply:
The team visibility setting affects whether those outside of the team _but_ within the organization can see the existence of the team. It does not allow _non-members_ (the community) to see the teams of the organization, which means an external list of teams must be made available for the community to view (either maintained manually by a Quilt staffer or automatically by a bot/automated process).

Reply:
https://quilt.gareth-coles.dev/about/teams/ has an updated list of all of the teams as they stand at this point in time, this will be merged into the master branch of the website soon enough:tm:

Reply:
```suggestion
[Technical Teams](#technical-teams) is outdated, but *all* teams can be found at https://quiltmc.org/about/teams/.
```